ui/src/main/java/edu/wpi/grip/ui/preview/PreviewsView.java

```diff
@@ -28,12 +35,15 @@
 
     private final EventBus eventBus;
     private final List<OutputSocket<?>> previewedSockets;
+    private final PipelineView pipeline;//This is used to determine the order the previews are shown in.
 
-    public PreviewsView(EventBus eventBus) {
+    public PreviewsView(EventBus eventBus, PipelineView pipeline) {
```

Comment:
So this is an issue with the original code, not my changes. Is this still something you would like me to do? Shouldn't it be in a separate pull request?